x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/TransportExplainLifecycleAction.java

```diff
@@ -109,7 +109,7 @@ protected void doMasterOperation(ExplainLifecycleRequest request, String[] concr
                 // If this is requesting only errors, only include indices in the error step or which are using a nonexistent policy
                 if (request.onlyErrors() == false
                     || (ErrorStep.NAME.equals(lifecycleState.getStep()) || indexLifecycleService.policyExists(policyName) == false)) {
-                    Long originationDate = idxSettings.getAsLong(LIFECYCLE_ORIGINATION_DATE, -1L);
+                    Long originationDate = idxMetadata.getSettings().getAsLong(LIFECYCLE_ORIGINATION_DATE, -1L);
```

Comment:
Is this change to fix something or was it inadvertent after undoing the other changes here?

Reply:
Yes "manual revert" from the draft. Good catch